CPS-0STL/README.md

```diff
@@ -0,0 +1,158 @@
+> | Use cases           | A concrete set of examples written from a user's perspective, describing what and why they are trying to do. When they exist, this section should give a sense of the current alternatives and highlight why they are unsuitable.                                                                                                                                                                                                                                                                                  |
+> | Goals               | A list of goals and non-goals a project is pursuing, ranked by importance. These goals should help understand the design space for the solution and what the underlying project is ultimately trying to achieve. <br/><br/>Goals may also contain requirements for the project. For example, they may include anything from a deadline to a budget (in terms of complexity or time) to security concerns. <br/><br/>Finally, goals may also serve as evaluation metrics to assess how good a proposed solution is. |
+> | Open Questions      | A set of questions to which any proposed solution should find an answer. Questions should help guide solutions design by highlighting some foreseen vulnerabilities or design flaws. Solutions in the form of CIP should thereby include these questions as part of their _'Rationale'_ section and provide an argued answer to each.                                                                                                                                                                              |
+> | _optional sections_ | If necessary, these sections may also be included in any order:<br/>**References**<br/>**Appendices**<br/>**Acknowledgements**<br>Do not add material in an optional section if it pertains to one of the standard sections.                                                                                                                                                                                                                                                                                       |
+> | Copyright           | The CPS must be explicitly licensed under acceptable copyright terms.                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+
```

Comment:
```suggestion
```
All of this should be removed unless the document is taken back to `Draft` review status (I can see all these sections have been filled out in the current document).  There is no H1 level header in either the CIP or CPS since this markup is generated from the YAML in the header.